### ISSUE TITLE
Training produces NaN loss and black sample images

### ISSUE THREAD

**Issue by 9wow** [gazingstars123/Anima-Standalone-Trainer#61]
When I run the training all the sample images are black and the training steps have "avr_loss=nan" from step 0-end. I've tried the default settings, custom settings, and a dummy dataset of 5 1024x1024 jpeg images with a caption but nothing seems to work. There are no errors in the console and everything loads normally (DiT, TE, VAE, dataset..).
Not sure how I can get this to work, AI cant figure it out either without them changing whats in the venv.
___
Windows 11
NVIDIA RTX 5090
Used specified torch/pytorch ("pip install torch==2.7.0 torchvision==0.22.0 --index-url https://download.pytorch.org/whl/cu128")

**Comment by 9wow** [comment]
Fixed by using pyenv to set the python to 3.12.11 and manually creating the venv + installing requirements. 
The install script ignores which python youre using if you have an unsupported version installed.
